docs/development/local_setup.md

```diff
@@ -69,31 +105,76 @@ $ git clone git@github.com:gardener/gardener.git
 $ cd gardener
 ```
 
-# Local development
+## Start the Gardener
 
 :warning: Before you start developing, please ensure to comply with the following requirements:
 
 1. You have understood the [principles of Kubernetes](https://kubernetes.io/docs/concepts/), and its [components](https://kubernetes.io/docs/concepts/overview/components/), what their purpose is and how they interact with each other.
 1. You have understood the [architecture of the Garden](https://github.com/gardener/documentation/wiki/Architecture), and what the various clusters are used for.
 
-The development of the Gardener could happen by targetting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Gardener controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
+The development of the Gardener could happen by targetting any cluster. You basically need a Garden cluster (e.g., a [Minikube](https://github.com/kubernetes/minikube) cluster) and one Seed cluster per cloud provider and per data center/region. You can configure the Garden controller manager to watch **all namespaces** for Shoot manifests or to only watch **one single** namespace.
 
-In order to ensure that a specific Seed cluster will be chosen, add the `.spec.cloud.seed` field (see [here](../../example/shoot-azure.yaml#L10) for an example Shoot manifest).
+The commands bellow will set your `minikube` with the absolute minimum resources to launch Gardener apiserver & controller manager:
 
-## Get started
+```bash
+$ minikube start
+Starting local Kubernetes v1.8.0 cluster...
+Starting VM...
+Getting VM IP address...
+Moving files into cluster...
+Setting up certs...
+Connecting to cluster...
+Setting up kubeconfig...
+Starting cluster components...
+Kubectl is now configured to use the cluster.
+Loading cached images from config file.
+
+$ make dev-setup
+clusterrolebinding "kube-dns-rule" created
+namespace "garden" created
+secret "default-domain" created
+secret "internal-domain-example-com" created
+deployment "etcd" created
+service "etcd" created
+```
 
-Please take a look at the [example manifests folder](../../example) to see which resource objects you need to install into your Garden cluster.
+Next, you need to run the API server and the Gardener controller manager using rules in the `Makefile`.
```

Comment:
Please prefix "API server" with "Gardener" to make it clear with API server is meant.

Reply:
maybe mention which rules in the Makefile should be used ?

Reply:
This is shown in the next line

Reply:
ah, ok, I was looking [here](https://github.com/gardener/gardener/pull/73) and couldn't see these lines there, Github UI is confusing ...